Jenkinsfile

```diff
@@ -203,82 +162,16 @@ pipeline {
             }
           }
         }
-        /**
```

Comment:
we remove those integration test steps, the job is not ready and would differ from the current definition so make no sense to have it here until it is not ready